Description: This patch is added to fix
 syntax errors on arithmetic expressions.

Forwarded: Yes
Bug-Debian: http://bugs.debian.org/551270
Author: Josue Abarca <jmaslibre@debian.org.gt>
Last-Update: 2010-05-21

--- a/utils/bombardment.in
+++ b/utils/bombardment.in
@@ -50,7 +50,7 @@
 sub=$inc*$numruns
 total=$total+$sub
 
-if [ 600 -lt $total ]
+if [ 600 -lt $(($total)) ]
 	then
 		echo "The options you specified will generate more than \
 600 concurrent users.  This causes problems on many systems.  Press \
@@ -96,8 +96,8 @@
 		echo "Starting run number" $j
 		%_PREFIX%/siege -f $site -c $currentcl -t $numurls -d $delay >> siege.$serial
 		currentcl=$currentcl+$inc
-		i=$i+1
-		j=$i+1
+		i=$(($i+1))
+		j=$(($i+1))
 		sleep 30
 	done
 
